src/index.ts

```diff
@@ -58,6 +58,7 @@ export * from './components/organisms/Form';
 export * from './components/organisms/Accordion';
 export { default as Navbar, navbarLinkStyles } from './components/organisms/Navbar/';
 export { default as Card } from './components/organisms/Card';
+export * from './components/organisms/Tabs';
```

Comment:
I have no idea where to add it in order for it to actually show up.

Reply:
If it's an organism or template you have to manually add the section in `/styleguide.config` follow the same pattern as the rest of the organisms.